*** .//smali/android/provider/Settings$System.smali	2014-06-23 03:13:56.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework.jar.out/.//smali/android/provider/Settings$System.smali	2014-06-23 03:11:37.000000000 +0300
***************
*** 140,145 ****
--- 140,147 ----
  
  .field public static final EGG_MODE:Ljava/lang/String; = "egg_mode"
  
+ .field public static final EMERGENCY_TONE:Ljava/lang/String; = "emergency_tone"
+ 
  .field public static final END_BUTTON_BEHAVIOR:Ljava/lang/String; = "end_button_behavior"
  
  .field public static final END_BUTTON_BEHAVIOR_DEFAULT:I = 0x2
