*** .//smali/com/android/internal/widget/ScrollingTabContainerView.smali	2014-06-23 03:14:41.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework2.jar.out/.//smali/com/android/internal/widget/ScrollingTabContainerView.smali	2014-06-23 03:12:18.000000000 +0300
***************
*** 667,672 ****
--- 667,690 ----
      goto :goto_0
  .end method
  
+ .method protected getSelectedTabIndex()I
+     .locals 1
+ 
+     .prologue
+     iget v0, p0, Lcom/android/internal/widget/ScrollingTabContainerView;->mSelectedTabIndex:I
+ 
+     return v0
+ .end method
+ 
+ .method protected getTabLayout()Landroid/widget/LinearLayout;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/internal/widget/ScrollingTabContainerView;->mTabLayout:Landroid/widget/LinearLayout;
+ 
+     return-object v0
+ .end method
+ 
  .method public onAttachedToWindow()V
      .locals 1
  
