*** .//smali/com/android/internal/widget/ScrollingTabContainerView$TabClickListener.smali	2014-06-23 03:14:41.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework2.jar.out/.//smali/com/android/internal/widget/ScrollingTabContainerView$TabClickListener.smali	2014-06-23 03:12:18.000000000 +0300
***************
*** 111,115 ****
--- 111,119 ----
  
      .end local v0           #child:Landroid/view/View;
      :cond_1
+     iget-object v4, p0, Lcom/android/internal/widget/ScrollingTabContainerView$TabClickListener;->this$0:Lcom/android/internal/widget/ScrollingTabContainerView;
+ 
+     invoke-static {v4, p1}, Lcom/android/internal/widget/Injector$ScrollingTabContainerViewHook;->onTabViewClick(Lcom/android/internal/widget/ScrollingTabContainerView;Landroid/view/View;)V
+ 
      return-void
  .end method
