*** .//smali/com/android/internal/telephony/gsm/SmsMessage.smali	2014-06-23 03:15:08.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/telephony-common.jar.out/.//smali/com/android/internal/telephony/gsm/SmsMessage.smali	2014-06-23 03:12:45.000000000 +0300
***************
*** 2124,2129 ****
--- 2124,2152 ----
      return v0
  .end method
  
+ .method public getRecipientAddress()Ljava/lang/String;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/internal/telephony/gsm/SmsMessage;->mRecipientAddress:Lcom/android/internal/telephony/gsm/GsmSmsAddress;
+ 
+     if-eqz v0, :cond_0
+ 
+     iget-object v0, p0, Lcom/android/internal/telephony/gsm/SmsMessage;->mRecipientAddress:Lcom/android/internal/telephony/gsm/GsmSmsAddress;
+ 
+     invoke-virtual {v0}, Lcom/android/internal/telephony/gsm/GsmSmsAddress;->getAddressString()Ljava/lang/String;
+ 
+     move-result-object v0
+ 
+     :goto_0
+     return-object v0
+ 
+     :cond_0
+     const/4 v0, 0x0
+ 
+     goto :goto_0
+ .end method
+ 
  .method public getStatus()I
      .locals 1
  
