*** .//smali/android/view/WindowManagerPolicy.smali	2014-06-23 03:14:19.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework.jar.out/.//smali/android/view/WindowManagerPolicy.smali	2014-06-23 03:11:57.000000000 +0300
***************
*** 23,28 ****
--- 23,30 ----
  
  .field public static final ACTION_PASS_TO_USER:I = 0x1
  
+ .field public static final ACTION_POKE_USER_ACTIVITY:I = 0x2
+ 
  .field public static final ACTION_WAKE_UP:I = 0x2
  
  .field public static final EXTRA_HDMI_PLUGGED_STATE:Ljava/lang/String; = "state"
