*** .//smali/com/android/internal/telephony/cdma/CdmaServiceStateTracker.smali	2014-06-23 03:15:06.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/telephony-common.jar.out/.//smali/com/android/internal/telephony/cdma/CdmaServiceStateTracker.smali	2014-06-23 03:12:43.000000000 +0300
***************
*** 8523,8528 ****
--- 8523,8532 ----
      move-result-object v1
  
      .local v1, plmn:Ljava/lang/String;
+     invoke-static {p0, v1}, Lcom/android/internal/telephony/Injector$ServiceStateTrackerHook;->getSpn(Lcom/android/internal/telephony/ServiceStateTracker;Ljava/lang/String;)Ljava/lang/String;
+ 
+     move-result-object v1
+ 
      iget-object v5, p0, Lcom/android/internal/telephony/cdma/CdmaServiceStateTracker;->mCurPlmn:Ljava/lang/String;
  
      invoke-static {v1, v5}, Landroid/text/TextUtils;->equals(Ljava/lang/CharSequence;Ljava/lang/CharSequence;)Z
