*** .//smali/android/hardware/SensorManager.smali	2014-06-23 03:13:21.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework.jar.out/.//smali/android/hardware/SensorManager.smali	2014-06-23 03:11:06.000000000 +0300
***************
*** 229,234 ****
--- 229,238 ----
      .parameter "p"
  
      .prologue
+     invoke-static {p0}, Landroid/hardware/Injector$SensorManagerHook;->getSeaLevelPressure(F)F
+ 
+     move-result p0
+ 
      const v0, 0x3e42dcae
  
      .local v0, coef:F
***************
*** 2332,2337 ****
--- 2336,2351 ----
  
  
  # virtual methods
+ .method public calibrateSensor(Landroid/hardware/Sensor;)Z
+     .locals 1
+     .parameter "sensor"
+ 
+     .prologue
+     const/4 v0, 0x0
+ 
+     return v0
+ .end method
+ 
  .method public cancelTriggerSensor(Landroid/hardware/TriggerEventListener;Landroid/hardware/Sensor;)Z
      .locals 1
      .parameter "listener"
