*** .//smali/com/android/internal/telephony/IccPhoneBookInterfaceManagerProxy.smali	2014-06-23 03:15:08.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/telephony-common.jar.out/.//smali/com/android/internal/telephony/IccPhoneBookInterfaceManagerProxy.smali	2014-06-23 03:12:45.000000000 +0300
***************
*** 72,77 ****
--- 72,86 ----
      return-object v0
  .end method
  
+ .method getIccPhoneBookInterfaceManager()Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Lcom/android/internal/telephony/IccPhoneBookInterfaceManagerProxy;->mIccPhoneBookInterfaceManager:Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;
+ 
+     return-object v0
+ .end method
+ 
  .method public setmIccPhoneBookInterfaceManager(Lcom/android/internal/telephony/IccPhoneBookInterfaceManager;)V
      .locals 0
      .parameter "iccPhoneBookInterfaceManager"
