*** .//smali/android/widget/Toast.smali	2014-06-23 03:14:29.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework.jar.out/.//smali/android/widget/Toast.smali	2014-06-23 03:12:07.000000000 +0300
***************
*** 405,410 ****
--- 405,422 ----
      return-void
  .end method
  
+ .method public setType(I)V
+     .locals 1
+     .parameter "type"
+ 
+     .prologue
+     iget-object v0, p0, Landroid/widget/Toast;->mTN:Landroid/widget/Toast$TN;
+ 
+     invoke-virtual {v0, p1}, Landroid/widget/Toast$TN;->setType(I)V
+ 
+     return-void
+ .end method
+ 
  .method public setView(Landroid/view/View;)V
      .locals 0
      .parameter "view"
