*** .//smali/android/speech/tts/TextToSpeech.smali	2014-06-23 03:14:02.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework.jar.out/.//smali/android/speech/tts/TextToSpeech.smali	2014-06-23 03:11:42.000000000 +0300
***************
*** 1934,1936 ****
--- 1934,1966 ----
  
      return v0
  .end method
+ 
+ .method public tryReconnectIfDead()V
+     .locals 1
+ 
+     .prologue
+     iget-object v0, p0, Landroid/speech/tts/TextToSpeech;->mServiceConnection:Landroid/speech/tts/TextToSpeech$Connection;
+ 
+     if-eqz v0, :cond_0
+ 
+     iget-object v0, p0, Landroid/speech/tts/TextToSpeech;->mServiceConnection:Landroid/speech/tts/TextToSpeech$Connection;
+ 
+     invoke-virtual {v0}, Landroid/speech/tts/TextToSpeech$Connection;->isConnected()Z
+ 
+     move-result v0
+ 
+     if-eqz v0, :cond_1
+ 
+     :cond_0
+     iget-object v0, p0, Landroid/speech/tts/TextToSpeech;->mServiceConnection:Landroid/speech/tts/TextToSpeech$Connection;
+ 
+     if-nez v0, :cond_2
+ 
+     :cond_1
+     invoke-virtual {p0}, Landroid/speech/tts/TextToSpeech;->shutdown()V
+ 
+     invoke-direct {p0}, Landroid/speech/tts/TextToSpeech;->initTts()I
+ 
+     :cond_2
+     return-void
+ .end method
