*** .//smali/android/os/PowerManager.smali	2014-06-23 03:13:46.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework.jar.out/.//smali/android/os/PowerManager.smali	2014-06-23 03:11:28.000000000 +0300
***************
*** 14,19 ****
--- 14,21 ----
  # static fields
  .field public static final ACQUIRE_CAUSES_WAKEUP:I = 0x10000000
  
+ .field public static final BRIGHTNESS_DIM:I = 0x14
+ 
  .field public static final BRIGHTNESS_OFF:I = 0x0
  
  .field public static final BRIGHTNESS_ON:I = 0xff
