*** .//smali/android/net/wifi/WifiMonitor$MonitorThread.smali	2014-06-23 03:13:39.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework.jar.out/.//smali/android/net/wifi/WifiMonitor$MonitorThread.smali	2014-06-23 03:11:23.000000000 +0300
***************
*** 100,105 ****
--- 100,129 ----
      return v5
  
      :cond_1
+     const-string v6, "WAPI:"
+ 
+     invoke-virtual {p1, v6}, Ljava/lang/String;->startsWith(Ljava/lang/String;)Z
+ 
+     move-result v6
+ 
+     if-eqz v6, :cond_miui
+ 
+     const-string v6, "pre-shared key may be incorrect"
+ 
+     invoke-virtual {p1, v6}, Ljava/lang/String;->indexOf(Ljava/lang/String;)I
+ 
+     move-result v6
+ 
+     if-lez v6, :cond_miui
+ 
+     iget-object v6, p0, Landroid/net/wifi/WifiMonitor$MonitorThread;->mStateMachine:Lcom/android/internal/util/StateMachine;
+ 
+     invoke-virtual {v6, v9}, Lcom/android/internal/util/StateMachine;->sendMessage(I)V
+ 
+     goto :goto_0
+ 
+     :cond_miui
+ 
      const-string v6, "WPS-SUCCESS"
  
      invoke-virtual {p1, v6}, Ljava/lang/String;->startsWith(Ljava/lang/String;)Z
