*** .//smali/android/graphics/drawable/DrawableContainer$DrawableContainerState.smali	2014-06-23 03:13:15.000000000 +0300
--- /Users/troj80/patchrom/patchrom_vivo/temp/new_smali/framework.jar.out/.//smali/android/graphics/drawable/DrawableContainer$DrawableContainerState.smali	2014-06-23 03:10:59.000000000 +0300
***************
*** 985,990 ****
--- 985,999 ----
      goto :goto_0
  .end method
  
+ .method public final getVariablePadding()Z
+     .locals 1
+ 
+     .prologue
+     iget-boolean v0, p0, Landroid/graphics/drawable/DrawableContainer$DrawableContainerState;->mVariablePadding:Z
+ 
+     return v0
+ .end method
+ 
  .method public growArray(II)V
      .locals 3
      .parameter "oldSize"
